src/locales/index.ts

```diff
@@ -1,9 +1,10 @@
 import { i18n } from "@lingui/core";
 import { en, fr, ko, tr, zh, ar, es, vi } from "make-plural/plurals";
+import { OHMLocaleSwitcherProps } from "@olympusdao/component-library";
 
 // Declare locales
 interface ILocale {
-  flag: string;
+  flag: OHMLocaleSwitcherProps["locales"]["locale"]["flag"];
```

Comment:
There was a semi breaking change with an update to the LocaleSwitcher component in component-library.  It's an unrelated change to the rest of the PR, but it resolves that issue.